### ISSUE TITLE
Unicode encoding bug

### ISSUE THREAD

**Issue by sumdog** [OliverUv/quickswitch-for-i3#4]
Quickswitch is awesome btw. Totally love it.

Anyway, quickswitch occasionally stops working. If I run it by hand, I get this stack trace:

```
Traceback (most recent call last):
  File "/home/sumit/bin/quickswitch.py", line 413, in <module>
    main()
  File "/home/sumit/bin/quickswitch.py", line 397, in main
    lookup = lookup_func()
  File "/home/sumit/bin/quickswitch.py", line 66, in get_windows
    return create_lookup_table(windows)
  File "/home/sumit/bin/quickswitch.py", line 205, in create_lookup_table
    rename_nonunique(windows)
  File "/home/sumit/bin/quickswitch.py", line 229, in rename_nonunique
    window_names[index] = "{} [{}]".format(name, i + 1)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2014' in position 34: ordinal not in range(128)
```

So it's when a window name has some funky Unicode character. I added a print statement and, in this instance, it was the following:

`Java Server Application Archetype — sbt-native-packager 1.0.0-M4 documentation - Mozilla Firefox`

They use that funky unicode dash in the title tag in their documentation! Savages! 

This seems to fix it:

`window_names[index] = "{} [{}]".format(name.decode('utf-8'), i + 1)`

Not sure if there are any other consequences of that. Will keep testing it.

**Comment by OliverUv** [comment]
Hi! Looks good and I appreciate your pull request! Do you want to keep testing this and report if you run into problems? If I haven't heard anything for 1-2 weeks, I'll pull it into master.

thanks :)

**Comment by OliverUv** [comment]
Fixed by sumdog

**Comment by lasers** [comment]
Hi all. I'm experiencing occasionally-broken quickswitch.py from time to time. I ran into one just now and wanted to investigate. I got this.

tchris ~ $ quickswitch.py --move
Traceback (most recent call last):
  File "/usr/bin/quickswitch.py", line 433, in <module>
    main()
  File "/usr/bin/quickswitch.py", line 418, in main
    lookup = lookup_func()
  File "/usr/bin/quickswitch.py", line 65, in get_windows
    return create_lookup_table(windows)
  File "/usr/bin/quickswitch.py", line 217, in create_lookup_table
    rename_nonunique(windows)
  File "/usr/bin/quickswitch.py", line 240, in rename_nonunique
    window_names[index] = "{} [{}]".format(name.decode('utf-8'), i + 1)
AttributeError: 'str' object has no attribute 'decode'                                                                                                       

Internet seems to tell me that I'm trying to decode an object that is already decoded since now we're on Python 3 where all strings are unicode objects.
